internal/lsp/protocol/tsprotocol.go

```diff
@@ -6750,6 +6750,7 @@ const (
 	LangXML             LanguageKind = "xml"
 	LangXSL             LanguageKind = "xsl"
 	LangYAML            LanguageKind = "yaml"
+	LangZig             LanguageKind = "zig"
```

Comment:
fwiw this file is generated

@kujtimiihoxha its not clear to me how we gen this, I guess its fine to manually add zig here, but i wonder why it wasn't here before...

Reply:
well... header says its generated hehe